operations/helm/charts/mimir-distributed/templates/metamonitoring/recording-rules.yaml

```
@@ -0,0 +1,5 @@
+{{- with .Values.metaMonitoring.prometheusRule.recording_rules }}
```

I think you meant to say 
```suggestion
{{- with .Values.metaMonitoring.prometheusRule.alerting_rules }}
```